swap/swap.go

```diff
@@ -132,11 +132,15 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 
 //GetPeerBalance returns the balance for a given peer
 func (swap *Swap) GetPeerBalance(peer enode.ID) (int64, error) {
+	fmt.Println("aa4aa")
```

Comment:
Please remove all these prints. I believe, you had an older version of the file before Fabio removed this printlns.

Reply:
Will do.

Reply:
done